### DISCUSSION TITLE
～ shows up with にょろ furigana/reading

### DISCUSSION THREAD

**Discussion by Morgawr** [Jitendex/Jitendex#28] [Errors and Corrections]
I'm not entirely sure if this can/should be considered a mistake or not as it depends on the interpretation but I feel like it is a mistake or at least inconsistent with how we handle punctuation everywhere else.

To show you what I mean, take for example this sentence as it shows up in the yomitan sentence search page: 

![GuAnWyNVkx](https://github.com/stephenmk/Jitendex/assets/184056/dc71e91d-8d92-4a6b-a175-ff62e4188433)

Yomitan adds furigana on top of any word it finds based on the dictionary entry (it is a bit hit or miss though).

Jmdict seems to have an "alternative" reading for ～ as [にょろ] (at least according to [jisho's entry](https://jisho.org/search/%EF%BD%9E), I didn't check the actual jmdict file) and this is likely why it gets picked up by yomitan's furigana. 

Other punctuation like 。 or 、 do not show up as まる or てん, but ～ shows up as にょろ. This to me feels a bit inconsistent.

Do you think this is useful/working as intended as is? Or should the にょろ reading be removed?

**Comment by stephenmk** [comment]
I don't like it either. I just submitted [an edit](https://www.edrdg.org/jmwsgi/entr.py?svc=jmdict&sid=&q=2415870.1) to drop the symbols from the kanji fields of the entry. We'll see if the other JMdict editors agree.

**Reply by stephenmk** [reply]
We've settled on removing the "〜" symbols as kanji forms for にょろ. Next version of jitendex will just have にょろ as its own headword.
